Doc/library/os.rst

```diff
@@ -193,6 +193,10 @@ process and user.
    to the environment made after this time are not reflected in :data:`os.environ`,
    except for changes made by modifying :data:`os.environ` directly.
 
+   The :meth:`!os.environ.refresh()` method updates :data:`os.environ` with
```

Comment:
FTR, the `:meth:` and `:func:` roles will implicitly add parentheses in the rendered output; you do not need to add them explicitly.